Regressed by https://github.com/unicode-org/icu/commit/c3fe7e09d844

collationcmds.c:467:51: error: use of undeclared identifier 'TRUE'
        uloc_toLanguageTag(localename, buf, sizeof(buf), TRUE, &status);
                                                         ^

--- src/backend/commands/collationcmds.c.orig	2020-08-10 21:17:40 UTC
+++ src/backend/commands/collationcmds.c
@@ -444,7 +444,7 @@ get_icu_language_tag(const char *localename)
 	UErrorCode	status;
 
 	status = U_ZERO_ERROR;
-	uloc_toLanguageTag(localename, buf, sizeof(buf), TRUE, &status);
+	uloc_toLanguageTag(localename, buf, sizeof(buf), true, &status);
 	if (U_FAILURE(status))
 		ereport(ERROR,
 				(errmsg("could not convert locale name \"%s\" to language tag: %s",
